### PR TITLE
Desktop portfolio

### PR DESCRIPTION
**Pull request by telanet**
Create a desktop version of Portfolio.
Use media queries in CSS file to make the desktop version look like the template selected (desktop width should be in 992px - 1100px range).
Use Grid and Flexbox lay out the elements on the page
Replicate the Figma design as much as possible

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Project Name
 
--Contact section of Mobile Portfolio
+-Desktop version of Portfolio
 
-![screenshot](./app_screenshot.png)
+!(./app_screenshot.png)
```
Comment (moise10r) [inline README.md]: - [ ] Kindly add a screenshot of your project to the README.md file to make it more professional

File: style.css [hunk 2]
```diff
@@ -1,841 +1,767 @@
...
 
-.logo {
-  font-size: 18px;
-  font-weight: 700;
-  color: #fff;
-  text-align: center;
+.horizontal-list {
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
```
Comment (moise10r) [inline style.css:31]: - [ OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
}

.flex-center {
  flex-direction: column;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

File: index.html [hunk 3]
```diff
@@ -9,382 +9,265 @@
...
+            <li class="list-item-text"><a href="#" class="link link-text">Contact</a></li>
+            <li class="list-item-text"><a href="#" class="link link-text fas fa-envelope"></a></li>
+          </ul>
+        <nav class="menu menu-icon" id="hamburger" >
+           <div class="icon-join">
+                <div class="bar bar1"></div>
+                <div class="bar bar2"></div>
+                <div class="bar bar3"></div>
+           </div>
       </nav>
```
Comment (moise10r) [inline index.html]: - [ ] Be careful with blank lines and indentation do not add blank lines, spaces, or indentations without a reason. Add two spaces of indentation to make your code more readable check the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: index.html [hunk 4]
```diff
@@ -9,382 +9,265 @@
...
+    <footer>
+        <hr class="h-rule">
+    <section class="bottom-navigation">
+        <ul class="social-media-icons ul-list horizontal-list">
+            <li class="list-item-icon"><a href="#" class="link link-icon fa fa-github"></a></li>
+            <li class="list-item-icon"><a href="#" class="link link-icon fa fa-linkedin"></a></li>
+            <li class="list-item-icon"><a href="#" class="link link-icon fa fa-angellist"></a></li>
+            <li class="list-item-icon"><a href="#" class="link link-icon fa fa-twitter"></a></li>
+            <li class="list-item-icon"><a href="#" class="link link-icon fab fa-medium-m"></a></li>
         </ul>
```
Comment (moise10r) [inline index.html]: - [ ] Please change the social-media-icons color and make position them to the center and not to the left, you must replicate the design as much as possible (e.g., font, colors, images, tests, margins) :point_down: 

Your Design:

s2

Figma Design:

s1

File: index.html [hunk 5]
```diff
@@ -9,382 +9,265 @@
...
+                    <li>
+                        <input type="text" id="name" placeholder="Full name" required maxlength="30">
+                    </li>
+                    <li>
+                        <input type="email" name="user-email" id="email" placeholder="Email address" required>
+                    </li>
+                    <li>
+                        <textarea name="comment" id="comment" placeholder="Enter text here" maxlength="500" required></textarea>
+                        <small id="error"></small>
+                        </li>
```
Comment (moise10r) [inline index.html]: - [ ] Please, Give the expected `width` to your `input` and `textarea` You need to make sure that your desktop version looks like the one in the Figma design template that you chose :point_down: 

Your Design:

des1


Figma Design:

des2

File: index.html [hunk 6]
```diff
@@ -9,382 +9,265 @@
...
+            <li class="list-item-text"><a href="#" class="link link-text">About</a></li>
+            <li class="list-item-text"><a href="#" class="link link-text">Contact</a></li>
+            <li class="list-item-text"><a href="#" class="link link-text fas fa-envelope"></a></li>
+          </ul>
+        <nav class="menu menu-icon" id="hamburger" >
+           <div class="icon-join">
+                <div class="bar bar1"></div>
+                <div class="bar bar2"></div>
+                <div class="bar bar3"></div>
+           </div>
```
Comment (moise10r) [inline index.html]: - [ ] Your hamburger menu does not display correctly, kindly fix it :point_down: 

hum

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@telanet,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- GitHubFlow was correctly used.  ✔️   
- Good commit history ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- [ ]  Your README looks great but you can improve it by making it more professional. A professional README file should have a project title, description, and instructions to set the project up locally. please refer to this [template](https://github.com/microverseinc/readme-template)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -9,382 +9,237 @@
     <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
```
Comment (rustam-tolipov) [inline index.html:9]: ## GOOD JOB 👏
- [ ] [OPTIONAL] You made an awesome design `^_+`. I have a suggestion for you to help make it better. To your all buttons set `cursor: pointer;` on hover and it changes cursor something like this `👆`

**Review by rustam-tolipov — APPROVED** [review]
## Hi @telanet 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**